cloud/blockstore/tools/testing/eternal_tests/eternal-load/bin/tests/test.py

```diff
@@ -13,9 +13,10 @@
 _REQUEST_SIZE = _REQUEST_BLOCK_COUNT * _BLOCKSIZE
 _REQUEST_COUNT = (_FILE_SIZE * 1024 ** 3) / _REQUEST_SIZE
 _BINARY_PATH = 'cloud/blockstore/tools/testing/eternal_tests/eternal-load/bin/eternal-load'
+_TIMEOUT = 30
 
 
-def __run_load(file_name):
+def __run_load_expect_fail(file_name):
```

Comment:
Why did the function name change? You have just added timeout parameter to run method.

Reply:
I would pretty much like to see a better naming after all. run load expect fail gives me a grasp of what does the function do, yet I can't fully comprehend it from the first sight.